### PR TITLE
Placement Refactor

### PR DESCRIPTION
**Pull request by nafiz1001**
image

### CODE REVIEW

**Comment by nafiz1001** [comment]
Do not fear the large diff. It's mostly package-lock.json 😉

**Comment by nafiz1001** [comment]
Should I get rid of cypress? The tests seem outdated.

File: frontend/src/reducers.js [hunk 1]
```diff
@@ -188,6 +189,7 @@ const allReducers = combineReducers({
   sampleNextStepTemplateActions,
   steps,
   notifications,
```
Comment (nafiz1001) [inline frontend/src/reducers.js:191]: i remember working on that, and I think no one uses that ;'(

**Comment by UlysseFG** [comment]
The list below the visualization do not put the selected samples first ... It makes it harder to know which samples were selected. I think the original version did that. 
The list is also randomly ordered making it very hard to find a sample using the list.
I would order the list by coordinates (as second ordering after the selection status).

**Comment by UlysseFG** [comment]
The popup while trying to place selected samples are distracting more than useful. It would be nice to deactivate popup when a sample are selected to be placed.
Also the popups should follow the currently used format container@coordinates or separate them (the wrapping of text in the popup is not great) instead of coordinates@container.
Sample freezeman ID in the popup is not useful (lab users will not know these). Sample name would be best (You could even include project). Of course, it is already better than nothing so do not remove the popup if these can't be fetched in an efficient manner.

**Comment by UlysseFG** [comment]
When loading an existing container, we need to block the coordinates that are already occupied by samples (and include a visual cue that those are taken and cannot be moved). At the moment, i could do placement to coordinates that are invalid.

**Review by UlysseFG — APPROVED** [review]

**Pull request merged**